Source/ZoomNet/Resources/CallLogs.cs

```diff
@@ -58,5 +58,41 @@ public Task<PaginatedResponseWithToken<UserCallLog>> GetAsync(string userId, Dat
 				.WithCancellationToken(cancellationToken)
 				.AsPaginatedResponseWithToken<UserCallLog>("call_logs");
 		}
+
+		/// <summary>
+		/// Get call logs for an entire account.
+		/// </summary>
+		/// <param name="from">The start date.</param>
+		/// <param name="to">The end date.</param>
+		/// <param name="type">Type of call log.</param>
+		/// <param name="pathType">Filter the API response by path of the call.</param>
+		/// <param name="recordsPerPage">The number of records to return.</param>
+		/// <param name="pagingToken">The paging token.</param>
+		/// <param name="siteId">Unique identifier of the site. Use this query parameter if you have enabled multiple sites and would like to filter the response of this API call by call logs of a specific phone site.</param>
+		/// <param name="chargedCallLogs">Whether to filter API responses to include call logs that only have a non-zero charge.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// An array of <see cref="AccountCallLog" />.
+		/// </returns>
+		public Task<PaginatedResponseWithTokenAndDateRange<AccountCallLog>> GetAsync(DateTime? from = null, DateTime? to = null, CallLogType type = CallLogType.All, CallLogPathType? pathType = null, int recordsPerPage = 30, string pagingToken = null, string siteId = null, bool chargedCallLogs = false, CancellationToken cancellationToken = default)
```

Comment:
Ah I didn't notice that recordsPerPage, pagingToken and cancellationToken were the last three. Love the consistency though. I will update that. Also I was on the fence about the start/end time because of parameter creep. it felt like it was getting long.  but one more wont hurt for completeness.

Reply:
I hear you regarding the large number of parameters. I have faced situations where I either felt there were too many parameters or some parameters did not apply in all situation. One way I have handled these situations is by creating separate methods like: CreateInstantMeetingAsync, CreateScheduledMeetingAsync and CreateRecurringMeetingAsync. It's documented as a single API endpoint, but not all parameter apply when creating a scheduled meeting vs a recuring meeting. That's why I created three separate methods each with the parameters that make sense.

Anyway, all this to say that in this case we have roughly a half-dozen parameters and I don't think adding an additional one is a big concern.